index.html

```diff
@@ -53,42 +54,16 @@ <h2 class="card-head">Tonic</h2>
         <li>css</li>
         <li>javaScript</li>
       </ul>
```

Comment:
- [ ] Kindly make sure the projects section is created dynamically with the information from the projects array as instructed by the previous reviewer, currently, only the popup window is created dynamically.

Reply:
- [ ] Kindly make sure all pop-up windows are showing,  the last two are not showing when I click on them below is a screenshot of the error am getting:
![image](https://user-images.githubusercontent.com/61408860/168379121-fe1ac8a5-5407-4002-bb8c-954dff0f44f2.png)